include/swift/AST/DiagnosticsFrontend.def

```diff
@@ -44,6 +44,11 @@ ERROR(error_unknown_library_level, none,
       "unknown library level '%0', "
       "expected one of 'api', 'spi', 'ipi', or 'other'", (StringRef))
 
+ERROR(error_old_spi_only_import_unsupported, none,
+      "'-experimental-spi-imports' is unsupported in Swift 6, "
+      "use '@_spiOnly' instead",
```

Comment:
I'm confused by the use of "instead" in this instruction. Wasn't the purpose of passing `-experimental-spi-imports` to allow your code to use `@_spiOnly import`? A project that already passes `-experimental-spi-imports` presumably already contains `@_spiOnly`.

Reply:
Would this be a reasonable alternative?
```
'@_spiOnly' no longer requires -experimental-spi-imports in the Swift 6 language mode
```

Reply:
Ahh ok yes that the source of my confusion.